lib/index.js

```diff
@@ -19,18 +19,21 @@ function spawnSeal(){
   }
 }
 
+function gravity (){
+  narwhal.y+=.5
+}
+
```

Comment:
This is the start of a gravity function. I think we should adjust this to degrade from forward momentum instead